Formula/prest.rb

```diff
@@ -0,0 +1,19 @@
+class Prest < Formula
+  desc "Serve a RESTful API from any PostgreSQL database"
+  homepage "https://github.com/nuveo/prest"
+  url "https://github.com/nuveo/prest/archive/v0.1.4.tar.gz"
+  sha256 "8d1a38e1ea45ebf7bffb34258ba2db5c404a58139597b6a8439bcaaeec12084e"
+
+  depends_on 'go' => :build
```

Comment:
Please use double quotes here; that’s what’s left to make the audit happy:
```
Error: 1 problem in 1 formula
prest:
  * C: 7: col 14: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```